generators/client-2/templates/src/main/webapp/app/blocks/interceptor/_http.interceptor.ts

```diff
@@ -31,7 +31,7 @@ export class HttpInterceptor extends Http {
     }
 
     request(url: string | Request, options?: RequestOptionsArgs): Observable<Response> {
-        return this.intercept(super.request(url, this.getRequestOptionArgs(options)));
+        return super.request(url, this.getRequestOptionArgs(options));
```

Comment:
Ok but I still dont get how we are doing it twice is there some other place where we call the intercept method? sorry didnt look into the code in detail yet